### PR TITLE
Scroll BlastForm to top when changing between views

### PR DESCRIPTION
**Pull request by msmanoj**
## Description
Scrolling the Blast form container to top when we switch between species and sequence view

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1549

## Deployment URL(s)
http:///blast-scroll-to-top.review.ensembl.org

## Views affected
BLAST -> species & sequences view on small screens

### CODE REVIEW

**Comment by azangru** [comment]
If you put the scrolling logic in `useEffect`, which reacts to the change in the step (sequences or species) in [here](https://github.com/Ensembl/ensembl-client/blob/dev/src/content/app/tools/blast/views/blast-form/BlastForm.tsx#L84), wouldn't it be both more idiomatic for React, and simpler to implement. You wouldn't need a document.querySelector; a useRef could give you exactly what you want there.

**Comment by msmanoj** [comment]
> If you put the scrolling logic in `useEffect`, which reacts to the change in the step (sequences or species) in [here](https://github.com/Ensembl/ensembl-client/blob/dev/src/content/app/tools/blast/views/blast-form/BlastForm.tsx#L84), wouldn't it be both more idiomatic for React, and simpler to implement. You wouldn't need a document.querySelector; a useRef could give you exactly what you want there; the main container will be within the immediate reach of your code.

That's done. A lot better now. Thanks

File: src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequencesHeader.tsx [hunk 1]
```diff
@@ -39,7 +39,6 @@ import type { SequenceType } from 'src/content/app/tools/blast/types/blastSettin
 
 import styles from './BlastInputSequences.scss';
 import sequenceBoxStyles from './BlastInputSequence.scss';
-
```
Comment (azangru) [inline src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequencesHeader.tsx:42]: could you undo this please? `git checkout dev src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequencesHeader.tsx`?

Comment (msmanoj) [reply src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequencesHeader.tsx]: That's done

**Review by azangru — APPROVED** [review]

**Pull request merged**